server/src/controllers/AchievementsController.js

```diff
@@ -0,0 +1,37 @@
+const Achievement = require('../models/Achievement');
+const replaceDriveURL = require('../utility/replaceDriveURL');
+
+module.exports = {
+  async createAchievement(req,res){
+    try{
+      const body = req.body;
+      let imageUrl = "";
+      if (!!body['Achievement Image'] && body['Achievement Image'].length > 0) {
+        imageUrl = replaceDriveURL(body['Achievement Image'][0]);
+      }
+      const achievement = {
+        title: body['Achievement Title'][0],
+        owner: {
+          fullName: body['Full Name'][0],
+          email: body['Email Address'][0]
+        },
+        projectUrl: body['Project URL (Optional)'][0]
```

Comment:
We are not sure if we will get this or not, since the form has an optional field for this. We can use the checking mechanism as you had done for the imageUrl.